https://bugs.gentoo.org/show_bug.cgi?id=468998

commit 0ee8d63649ab475fe2a16b2d62b5a5e2a8a69829
Author: Zbigniew JÄ™drzejewski-Szmek <zbyszek@in.waw.pl>
Date:   Tue May 7 11:35:23 2013 -0400

    systemd-sleep: it is not an error if the config file is missing

diff --git a/src/shared/sleep-config.c b/src/shared/sleep-config.c
index 73a3acb..cd3238b 100644
--- a/src/shared/sleep-config.c
+++ b/src/shared/sleep-config.c
@@ -47,19 +47,16 @@ int parse_sleep_config(const char *verb, char ***modes, char ***states) {
         FILE _cleanup_fclose_ *f;
 
         f = fopen(PKGSYSCONFDIR "/sleep.conf", "re");
-        if (!f) {
-                if (errno == ENOENT)
-                        return 0;
-
-                log_warning("Failed to open configuration file " PKGSYSCONFDIR "/sleep.conf: %m");
-                return 0;
+        if (!f)
+                log_full(errno == ENOENT ? LOG_DEBUG: LOG_WARNING,
+                         "Failed to open configuration file " PKGSYSCONFDIR "/sleep.conf: %m");
+        else {
+                r = config_parse(NULL, PKGSYSCONFDIR "/sleep.conf", f, "Sleep\0",
+                                 config_item_table_lookup, (void*) items, false, false, NULL);
+                if (r < 0)
+                        log_warning("Failed to parse configuration file: %s", strerror(-r));
         }
 
-        r = config_parse(NULL, PKGSYSCONFDIR "/sleep.conf", f, "Sleep\0",
-                         config_item_table_lookup, (void*) items, false, false, NULL);
-        if (r < 0)
-                log_warning("Failed to parse configuration file: %s", strerror(-r));
-
         if (streq(verb, "suspend")) {
                 /* empty by default */
                 *modes = suspend_mode;
